src/main/content/guides.html

```diff
@@ -178,6 +181,43 @@ <h3 class="guide-title">{{guide.title}}</h3>
     </div>
 {% endif %}
 
+<!-- Management Guides -->
```

Comment:
More guides should be the last on the page so put these above the "More Guides" section